components/board/genericlinux/data.go

```diff
@@ -144,53 +144,41 @@ func getPwmChipDefs(pinDefs []PinDefinition) (map[string]pwmChipData, error) {
 
 	// Now, look for all chips whose names we found.
 	pwmChipsInfo := map[string]pwmChipData{}
+	var found bool
+	const sysPrefix = "/sys/class/pwm"
```

Comment:
I'm content with the current name, but it's not great because this isn't really a prefix, and it's not obvious what the system in "sys" is. Maybe `sysfsDir`? Maybe `pwmDir`. Maybe `sysfsPwm`? Maybe leave as-is; none of my suggestions are _great._ :shrug: